services/vendors/Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL := help
+
+# Internal VARIABLES ------------------------------------------------
+# STACK_NAME defaults to name of the current directory. Should not to be changed if you follow GitOps operating procedures.
+STACK_NAME = $(notdir $(shell pwd))
+TEMP_COMPOSE=.stack.${STACK_NAME}.yaml
+REPO_BASE_DIR := $(shell git rev-parse --show-toplevel)
+
+# TARGETS --------------------------------------------------
+include ${REPO_BASE_DIR}/scripts/common.Makefile
+
+.PHONY: up ## Deploys registry stack
+up: ${TEMP_COMPOSE} .init .env
+	docker stack deploy --with-registry-auth --prune --compose-file $< ${STACK_NAME}
```

Comment:
is the indentation right here?

Reply:
Yes, although it looks indeed weird in "github" diff